packages/typespec-ts/test/integration/generated/mediaTypes/src/mediaTypesClient.ts

```diff
@@ -7,7 +7,7 @@ import { MediaTypesClient } from "./clientDefinitions";
 
 /**
  * Initialize a new instance of `MediaTypesClient`
- * @param $host - The parameter $host
+ * @param $host - A sequence of textual characters.
```

Comment:
Seems the description enriched successfully from type and I think it is expected.